docs/serverless/rules/alerts-ui-monitor.mdx

```diff
@@ -41,27 +41,60 @@ For detailed information on a rule, the alerts it generated, and associated erro
 
-![Rule execution results tab](../images/alerts-ui-monitor/-detections-rule-execution-logs.png)
+![Rule execution log on the rule execution results tab](../images/alerts-ui-monitor/-detections-rule-execution-logs.png)
 
 You can hover over each column heading to display a tooltip about that column's data. Click a column heading to sort the table by that column.
 
 Use these controls to filter what's included in the logs table:
 
+* The **Run type** drop-down filters the table by rule execution type: 
+    * **Scheduled**: Rule executions that the detection engine automatically ran according to the rule's schedule.
+    * **Manual**: Rule executions that were <DocLink slug="/serverless/security/rules-ui-management" section="manually-run-rules">started manually</DocLink>.
+
 * The **Status** drop-down filters the table by rule execution status: 
     * **Succeeded**: The rule completed its defined search. This doesn't necessarily mean it generated an alert, just that it ran without error.
     * **Failed**: The rule encountered an error that prevented it from running. For example, a ((ml)) rule whose corresponding ((ml)) job wasn't running.
     * **Warning**: Nothing prevented the rule from running, but it might have returned unexpected results. For example, a custom query rule tried to search an index pattern that couldn't be found in ((es)).
 
 * The date and time picker sets the time range of rule executions included in the table. This is separate from the global date and time picker at the top of the rule details page.
 
+* The **Source event time range** toggle includes more or less data in the table, pertaining to the time range of a manual run.
+
 * The **Show metrics columns** toggle includes more or less data in the table, pertaining to the timing of each rule execution.
 
 * The **Actions** column allows you to show alerts generated from a given rule execution. Click the filter icon (<DocIcon type="filterInCircle" title="Filter" size="s"/>) to create a global search filter based on the rule execution's ID value. This replaces any previously applied filters, changes the global date and time range to 24 hours before and after the rule execution, and displays a confirmation notification. You can revert this action by clicking **Restore previous filters** in the notification.
 
+<div id="manual-runs-table"></div>
+
+### Manual runs table
+
+<DocBadge template="technical preview" />
+
+Execution details of active manual runs are shown in the Manual runs table. Each manual run can produce multiple rule executions, depending on the time range of the run and the rule's execution schedule.
+
+To access the Manual runs table, go to **Rules** -> **Detection rules (SIEM)**, click the rule's name to open its details, then scroll down and select the **Execution results** tab. Scroll down again to find the Manual runs table. 
```

Comment:
Another opportunity to avoid platform-dependent nav steps. The Rules page is especially tricky because of how they've named it different things in different places, but maybe users would still understand something more generic:
```suggestion
To access the Manual runs table, navigate to the detection rules page, click the rule's name to open its details, then scroll down and select the **Execution results** tab. Scroll down again to find the Manual runs table. 
```